### DISCUSSION TITLE
Directly copy out of the Jbrowse

### DISCUSSION THREAD

**Discussion by kiprusov** [GMOD/jbrowse-components#4183] [Q&A] [answered]
Hi there, 

Just a short question: is it possible to copy a selected region to the browser directly if the Jbrowse is embedded. We basically need instead of putting the coordinates manually to be able to open a Jbrowse in popup, select a region, click (for example, under the "bookmark"), and the coordinates of the region will be transferred to the textbox on site while the popup will be closed. 

Is it feasible to do? 
Thanks!

**Comment by cmdcolin** [comment]
Can you make a mockup of this in MSpaint or something similar, or take a screenshot of jbrowse and point to it would work? Just from my understanding you want to copy the coordinates of a region after click-and-drag select (what we sometimes call rubberband select) but wanted to confirm with you

**Comment by kiprusov** [comment]
Sure! So that's what we need to do - or something like this;)

So a user in our interface can go to the Genome browser via a link under the text box, select a chromosome/position in a popup, click right on the region and get the sequence (or coordinates, it doesn't matter) into the application back.

![image](https://github.com/GMOD/jbrowse-components/assets/133758203/389924ff-0219-4fb5-8ac0-76a29a102aa6)
<img width="863" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/133758203/425f18ba-b696-443d-970f-0d33054d9333">
<img width="394" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/133758203/9e30bf11-e022-44c6-873f-c7cb7b9c6bf7">

**Reply by cmdcolin** [reply, marked as answer]
@kiprusov you can actually see an example of adding custom "rubberband menu items" in this storybook. if you are using embedded (it looks like you might be) then that storybook may make the process easier. this way, you can much more easily add your own custom menu items https://jbrowse.org/storybook/lgv/main/?path=/docs/using-plugins--docs

look at the "Source code" links on that page to see how they were done

screenshot showing the custom menu items

![image](https://github.com/GMOD/jbrowse-components/assets/6511937/cc68da1a-d518-489a-ba76-910306119599)

**Reply by cmdcolin** [reply]
we could also consider adding a behavior like this to the default set of menu items, but just wanted to make you aware of the ability to do custom things with these menu items

**Reply by kiprusov** [reply]
Thanks a lot!
